### DISCUSSION TITLE
V2 Roadmap

### DISCUSSION THREAD

**Discussion by cosmtrek** [air-verse/air#163] [General]
Never thought it's a project with thousands of stars, so I would like to develop new version named `v2`:)

There are some possible features according to issues:
* simplify air.yaml
* support powerful cli mode like [reflex](https://github.com/cespare/reflex)
* improve engine
* better terminal display
* support live reload protocol

Other features are welcome~

**Comment by kilisei** [comment]
Hello, is it still planned to do a v2, mainly for the `YAML` config file because most other tools use `YAML`, thanks. :)
